### PR TITLE
feat: dedicated package for thor solo accounts and constants

### PR DESCRIPTION
**Pull request by rodolfopietro97**
# Description

Create dedicated Thor solo package

# Checklist:

- [x] Create package
- [x] Thor solo constants
- [x] Built in constants
- [x] Network constant (WIP)
- [x] Check other constants
- [x] Thor solo doc/tutorial
- [x] Doc

### CODE REVIEW

File: packages/core/src/utils/txt/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './txt';
```
Comment (fabiorigam) [inline packages/core/src/utils/txt/index.ts:1]: What is this empty folder? do we need it?

File: packages/network/src/constant/built-in/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from './built-in';
```
Comment (fabiorigam) [inline packages/network/src/constant/built-in/index.ts:1]: Same here

**Review by fabiorigam — CHANGES_REQUESTED** [review]

File: packages/core/src/utils/txt/index.ts (hunk 1 above)
Comment (rodolfopietro97) [reply packages/core/src/utils/txt/index.ts:1]: This was a missing export (note related to this PR, but I saw this and I correct).
If you see into `packages/core/src/utils/txt/` folder you will find the file

File: packages/network/src/constant/built-in/index.ts (hunk 2 above)
Comment (rodolfopietro97) [reply packages/network/src/constant/built-in/index.ts:1]: Same here, see 'packages/network/src/constant/built-in' folder

**Review by fabiorigam — APPROVED** [review]

**Pull request merged**